typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
```

Comment:
That doesn't solve anything.  You have code that runs at module load time in both modules.  It doesn't matter how/what you load from the module, when you load it, all the global code runs.